internal/app/handle.go

```diff
@@ -85,3 +113,77 @@ func handleGokuDefault(w http.ResponseWriter, r *http.Request) {
 	// Writes the JSON bytes to the JSON response writer
 	w.Write(d)
 }
+
+// lenOfOne handles what JSON message to give back for a JSON array
+// with one fighter in it
+func lenOfOne(f fighter, a *allegiance) {
```

Comment:
since these functions will likely not be exclusive to this package, you should put them in their own package.